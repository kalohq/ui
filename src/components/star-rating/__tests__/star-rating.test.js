/* eslint-env jest */
import React from 'react';
import renderer from 'react-test-renderer';

import StarRating from '../star-rating.react';

describe('StarRating', () => {
  test('should render properly', () => {
    const wrapper = renderer.create(<StarRating score={3.7} />).toJSON();

    expect(wrapper).toMatchSnapshot();
  });
});
