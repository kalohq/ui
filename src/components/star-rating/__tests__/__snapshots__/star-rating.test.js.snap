// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StarRating should render properly 1`] = `
<div
  className="root e1krcyn16 css-0"
  style={Object {}}
>
  <div
    className="star"
    style={
      Object {
        "height": 20,
        "width": 20,
      }
    }
  >
    <span
      className="icon background"
    >
      <i
        className="ui-icon ui-icon--size-20 fill-currentColor e1krcyn16 css-0"
        onClick={undefined}
        style={Object {}}
        title={undefined}
      >
        <svg
          aria-hidden="true"
          height={20}
          width={20}
        >
          <use
            xlinkHref="#star"
          />
        </svg>
      </i>
    </span>
    <span
      className="icon foreground"
      style={
        Object {
          "color": "#f07a7a",
        }
      }
    >
      <i
        className="ui-icon ui-icon--size-20 fill-currentColor e1krcyn16 css-0"
        onClick={undefined}
        style={Object {}}
        title={undefined}
      >
        <svg
          aria-hidden="true"
          height={20}
          width={20}
        >
          <use
            xlinkHref="#star"
          />
        </svg>
      </i>
    </span>
  </div>
  <div
    className="star"
    style={
      Object {
        "height": 20,
        "width": 20,
      }
    }
  >
    <span
      className="icon background"
    >
      <i
        className="ui-icon ui-icon--size-20 fill-currentColor e1krcyn16 css-0"
        onClick={undefined}
        style={Object {}}
        title={undefined}
      >
        <svg
          aria-hidden="true"
          height={20}
          width={20}
        >
          <use
            xlinkHref="#star"
          />
        </svg>
      </i>
    </span>
    <span
      className="icon foreground"
      style={
        Object {
          "color": "#f07a7a",
        }
      }
    >
      <i
        className="ui-icon ui-icon--size-20 fill-currentColor e1krcyn16 css-0"
        onClick={undefined}
        style={Object {}}
        title={undefined}
      >
        <svg
          aria-hidden="true"
          height={20}
          width={20}
        >
          <use
            xlinkHref="#star"
          />
        </svg>
      </i>
    </span>
  </div>
  <div
    className="star"
    style={
      Object {
        "height": 20,
        "width": 20,
      }
    }
  >
    <span
      className="icon background"
    >
      <i
        className="ui-icon ui-icon--size-20 fill-currentColor e1krcyn16 css-0"
        onClick={undefined}
        style={Object {}}
        title={undefined}
      >
        <svg
          aria-hidden="true"
          height={20}
          width={20}
        >
          <use
            xlinkHref="#star"
          />
        </svg>
      </i>
    </span>
    <span
      className="icon foreground"
      style={
        Object {
          "color": "#f07a7a",
        }
      }
    >
      <i
        className="ui-icon ui-icon--size-20 fill-currentColor e1krcyn16 css-0"
        onClick={undefined}
        style={Object {}}
        title={undefined}
      >
        <svg
          aria-hidden="true"
          height={20}
          width={20}
        >
          <use
            xlinkHref="#star"
          />
        </svg>
      </i>
    </span>
  </div>
  <div
    className="star"
    style={
      Object {
        "height": 20,
        "width": 20,
      }
    }
  >
    <span
      className="icon background"
    >
      <i
        className="ui-icon ui-icon--size-20 fill-currentColor e1krcyn16 css-0"
        onClick={undefined}
        style={Object {}}
        title={undefined}
      >
        <svg
          aria-hidden="true"
          height={20}
          width={20}
        >
          <use
            xlinkHref="#star"
          />
        </svg>
      </i>
    </span>
    <span
      className="icon foreground half"
      style={
        Object {
          "color": "#f07a7a",
        }
      }
    >
      <i
        className="ui-icon ui-icon--size-20 fill-currentColor e1krcyn16 css-0"
        onClick={undefined}
        style={Object {}}
        title={undefined}
      >
        <svg
          aria-hidden="true"
          height={20}
          width={20}
        >
          <use
            xlinkHref="#star"
          />
        </svg>
      </i>
    </span>
  </div>
  <div
    className="star"
    style={
      Object {
        "height": 20,
        "width": 20,
      }
    }
  >
    <span
      className="icon background"
    >
      <i
        className="ui-icon ui-icon--size-20 fill-currentColor e1krcyn16 css-0"
        onClick={undefined}
        style={Object {}}
        title={undefined}
      >
        <svg
          aria-hidden="true"
          height={20}
          width={20}
        >
          <use
            xlinkHref="#star"
          />
        </svg>
      </i>
    </span>
    <span
      className="icon foreground"
      style={
        Object {
          "color": false,
        }
      }
    >
      <i
        className="ui-icon ui-icon--size-20 fill-currentColor e1krcyn16 css-0"
        onClick={undefined}
        style={Object {}}
        title={undefined}
      >
        <svg
          aria-hidden="true"
          height={20}
          width={20}
        >
          <use
            xlinkHref="#star"
          />
        </svg>
      </i>
    </span>
  </div>
</div>
`;
